packages/custom/icu/public/components/entity-list/regions/header.html

```diff
@@ -4,6 +4,6 @@
 >
     <header class="title">
         {{ entityName + 'From' | i18next}}{{currentContext.entityName | i18next}}
-        <span class="entityTitle" ng-bind-html="currentContext.entity.title ? currentContext.entity.title : currentContext.entity.name"></span>
```

Comment:
tried to solve this with context service, but as i see this service is unfinished, or works not like it should